src/redux/selectors/ui/get_active_market_article.js

```diff
@@ -0,0 +1,14 @@
+import _get from 'lodash/get'
+import { createSelector } from 'reselect'
+import { getActiveMarket } from '.'
+import { REDUCER_PATHS } from '../../config'
+
+const path = REDUCER_PATHS.UI
+const EMPTY_OBJ = {}
+
+const getActiveMarketArticle = createSelector([getActiveMarket, state => state], (activeMarket, state) => {
```

Comment:
using `state` as 2nd param does not make sense with createSelector as memoising whole state wont be helpful, instead use other selector that returns only ccyArticles
const getCcyArticles = (state) => _get(state, `${path}.ccyArticles, EMPTY_OBJ) and then use it as 2nd param to createSelector